bigquery/google/cloud/bigquery/magics.py

```diff
@@ -417,11 +417,20 @@ def _cell_magic(line, query):
     elif args.maximum_bytes_billed is not None:
         value = int(args.maximum_bytes_billed)
         job_config.maximum_bytes_billed = value
-    query_job = _run_query(client, query, job_config)
+
+    error = None
+    try:
+        query_job = _run_query(client, query, job_config)
+    except Exception as ex:
+        error = str(ex)
 
     if not args.verbose:
         display.clear_output()
 
+    if error:
+        print("\nERROR:\n", error)
```

Comment:
We should print to stderr so that it shows up as an error in the notebook view.

(May need `from __future__ import print_function`)